packages/opentelemetry-host-metrics/test/metric.test.ts

```diff
@@ -156,31 +163,31 @@ describe('Host Metrics', () => {
     it('should export CPU time metrics', async () => {
       const metric = await getRecords(reader, 'system.cpu.time');
 
-      ensureValue(metric, { state: 'user', cpu: '0' }, 90713.56);
-      ensureValue(metric, { state: 'system', cpu: '0' }, 63192.630000000005);
-      ensureValue(metric, { state: 'idle', cpu: '0' }, 374870.7);
+      ensureValue(metric, { state: 'user', cpu: '0' }, 90714.26);
+      ensureValue(metric, { state: 'system', cpu: '0' }, 63192.83);
+      ensureValue(metric, { state: 'idle', cpu: '0' }, 374870.8);
       ensureValue(metric, { state: 'interrupt', cpu: '0' }, 0);
       ensureValue(metric, { state: 'nice', cpu: '0' }, 0);
 
-      ensureValue(metric, { state: 'user', cpu: '1' }, 11005.42);
-      ensureValue(metric, { state: 'system', cpu: '1' }, 7678.12);
-      ensureValue(metric, { state: 'idle', cpu: '1' }, 510034.8);
+      ensureValue(metric, { state: 'user', cpu: '1' }, 11005.72);
+      ensureValue(metric, { state: 'system', cpu: '1' }, 7678.62);
+      ensureValue(metric, { state: 'idle', cpu: '1' }, 510035);
       ensureValue(metric, { state: 'interrupt', cpu: '1' }, 0);
       ensureValue(metric, { state: 'nice', cpu: '1' }, 0);
     });
 
     it('should export CPU utilization metrics', async () => {
       const metric = await getRecords(reader, 'system.cpu.utilization');
 
-      ensureValue(metric, { state: 'user', cpu: '0' }, 30247.935978659552);
-      ensureValue(metric, { state: 'system', cpu: '0' }, 21071.23374458153);
-      ensureValue(metric, { state: 'idle', cpu: '0' }, 124998.56618872957);
+      ensureValue(metric, { state: 'user', cpu: '0' }, 0.7);
```

Comment:
Is the value rounded or is this just a coincidence that they're all 1 significant figure?

Reply:
It's not a coincidence. The data is mocked with `packages/opentelemetry-host-metrics/test/mocks/cpu.json`.